model-optimizer/extensions/middle/layer_normalization.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from extensions.ops.elementwise import Add, Mul
+from extensions.ops.mvn import MVN
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.tf.graph_utils import create_op_node_with_second_input
+from mo.graph.graph import Graph
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.ops.squeeze import Squeeze
+from mo.ops.unsqueeze import Unsqueeze
+
+
+class LayerNormalization(MiddleReplacementPattern):
```

Comment:
Could you write the description of the operation?

Reply:
Below